packages/core/android/src/main/kotlin/com/datadog/reactnative/DatadogSDKWrapper.kt

```diff
@@ -70,6 +70,16 @@ object DatadogSDKWrapperStorage {
 }
 
 internal class DatadogSDKWrapper : DatadogWrapper {
+    internal var rumForLogsState = RUM_ENABLE_LOGS_DEFAULT
+    internal var traceForLogsState = TRACES_ENABLE_LOGS_DEFAULT
+
+    internal fun setRumForLogsEnabled(enabled: Boolean) {
+        this.rumForLogsState = enabled
+    }
+
+    internal fun setTracesForLogsEnabled(enabled: Boolean) {
+        this.traceForLogsState = enabled
+    }
```

Comment:
these methods seem redundant, you already have `var` properties with the same visibility, you can use them directly